Lib/xml/etree/ElementPath.py

```diff
@@ -72,24 +72,27 @@
 
 def xpath_tokenizer(pattern, namespaces=None):
     default_namespace = namespaces.get('') if namespaces else None
+    not_parsing_attribute = True
```

Comment:
Personally I'd use `parsing_attribute` here (and `not parsing_attribute` below), just because it's easier to read. But the logic looks fine to me